test/fixtures/tags.yml

```diff
@@ -78,4 +78,8 @@ co-author:
 
 unpublished-note-tag:
   tid: 20
-  name: unpublished-note-tag
\ No newline at end of file
+  name: unpublished-note-tag
+  
+ cool:
```

Comment:
Oh this has a leading space as well! Sorry, didn't catch it. Try this: 

```suggestion
cool:
```

Reply:
woops, thanks for pointing that out!